.github/workflows/release-airgap-artifacts.yaml

```diff
@@ -13,6 +13,11 @@ jobs:
     runs-on:
       - self-hosted
       - medium
+
+    env:
```

Comment:
it looks like we don’t have these explicitly set in the other release branches. should we backport the full PR and see if that works? 

i.e. https://github.com/mesosphere/dkp-catalog-applications/pull/107

Reply:
Absolutely! 💯